docs/proposals/27-project-group.md

```diff
@@ -0,0 +1,197 @@
+  projects:
+    - Project1
+    - Project2
+```
+
+The main idea behind `ProjectGroup`s is that they, like projects, have their own namespace.
+Every resource that is shareable (at first only `SeedBinding`s) in this namespace, will be shared to all projects in this project group.
+
+This sharing mechanism will be achieved by the `ProjectGroup` controller.
+It will copy the objects from the project group namespace to the namespaces of the projects that belong to the project group.
+
+When the object in the project group namespace is changed, the controller will immediately update the objects in the namespaces of the projects.
+
+To let a user in a project know that such an object is being kept in sync with an object in a project group, a `projectgroup.gardener.cloud/copied-from: <projectgroup-name>` label will be added to it.
+
+This copying feature can become useful for future use cases, for example for sharing infrastructure credentials across different projects, by copying `SecretBinding`s.
+
+It will not be possible to add non-existent project to a project group. This also means that if a project is deleted that is part of a project group, the controller should remove it.
+
+The `.spec.namespace` field in `ProjectGroup` will behave equally as the `Project` `.spec.namespace` field.
+
+##### Permissions
+
+Anybody should be able to create a project group.
+
+But adding a project to a project group is a sensitive operation.
+
+If anyone could add any project to their project group, a bad actor could add a project of a third party to their project group and restrict them from using any seed except the ones bound in the bad actors project group.
+
+To circumvent a scenario like this, a new role, called `project-group-assigner` will be introduced that allows people with this role to add the project to a project group, in which they have the same role.
+
+#### SeedBinding
+
+To enable the general functionality of the two scenarios described in the [Motivation](#motivation) in a singular concept, `SeedBinding`s shall be introduced.
+
+Seed bindings will be considered during the scheduling of a shoot, in a way that if a shoot belongs to a project with a seed binding, only seeds that are targeted by the seed binding will be selected for scheduling.
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: SeedBinding
+metadata:
+  name: privateseed-provider-1
+  namespace: garden-project1
+taintSeed: false
+seedSelector:
+  matchLabels:
+    foo: bar
+
+```
+
+If `.taintSeed` is set to `true`, the specified seed(s) will be tainted, so that no other projects outside the project group can schedule shoots on them.
+
+Projects that have such a tainting binding will receive the `.spec.tolerations.[]whitelist` and `.spec.tolerations.[]defaults` counterpart toleration by the seed binding controller.
+
+The name of the taint will be derived from its `.metadata.name`. With that, there will be no conflict when the seed binding gets shared to other namespaces.
```

Comment:
Should the taint name not be globally unique, i.e. does it need the namespace as well?